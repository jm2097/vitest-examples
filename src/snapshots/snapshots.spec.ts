import { describe, expect, it } from "vitest";
import { toUpperCase } from "./snapshots";

describe("snapshots", () => {
  it("toUpperCase", () => {
    const result = toUpperCase("foobar");
    expect(result).toMatchSnapshot();
  });

  it("toUpperCase", () => {
    const result = toUpperCase("foobar");
    expect(result).toMatchInlineSnapshot('"FOOBAR"');
  });
});
