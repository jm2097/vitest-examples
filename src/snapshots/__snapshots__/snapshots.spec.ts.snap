// Vitest Snapshot v1

exports[`snapshots > toUpperCase 1`] = `"FOOBAR"`;
